package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@seneca/telescope",
   "private": true,
-  "version": "0.8.2",
+  "version": "0.0.1",
```

Comment:
Let's not do this.  Can you roll back this commit?

Reply:
Sorry I checked out the wrong package.json should be fixed now